### ISSUE TITLE
Hoodie doesn't like spaces in path names

### ISSUE THREAD

**Issue by adambrault** [hoodiehq/bugs#5]
## Description

I was attempting to start an app server for the first time and it hung waiting on CouchDB then informed me that Couch timed out.

```
CouchDB started: http://127.0.0.1:6003
Waiting for CouchDB [---*--] FAILED 

ERR! Error: Timed out waiting for CouchDB. These logs may help:
ERR! 	/Users/adam/Dropbox (&yet)/Dev/encourage/data/couch.stderr
ERR! 	/Users/adam/Dropbox (&yet)/Dev/encourage/data/couch.stdout
ERR! 	/Users/adam/Dropbox (&yet)/Dev/encourage/data/couch.log
ERR!     at Request._callback (/Users/adam/Dropbox (&yet)/Dev/encourage/node_modules/hoodie-server/lib/couchdb/index.js:145:27)
ERR!     at self.callback (/Users/adam/Dropbox (&yet)/Dev/encourage/node_modules/hoodie-server/node_modules/request/request.js:373:22)
ERR!     at Request.EventEmitter.emit (events.js:95:17)
ERR!     at Request.onRequestError (/Users/adam/Dropbox (&yet)/Dev/encourage/node_modules/hoodie-server/node_modules/request/request.js:971:8)
ERR!     at ClientRequest.EventEmitter.emit (events.js:95:17)
ERR!     at Socket.socketErrorListener (http.js:1547:9)
ERR!     at Socket.EventEmitter.emit (events.js:95:17)
ERR!     at net.js:440:14
ERR!     at process._tickDomainCallback (node.js:459:13)
```

You'll note that I have my path weirdly set up as ``/Users/adam/Dropbox (&yet)/Dev/encourage``. (DRAMATIC FORESHADOWING!)

Annoyingly, a year ago, Dropbox forced business user folders to be renamed from "Dropbox" to two separate folders: "Dropbox (Personal)" and "Dropbox (Company Name)" with no option to change that. (I know, ugh, Dropbox.) I regularly use two computers and it's nice to have them automatically synced, so I have a ~/Dev folder that is symlinked to a Dev folder in my Dropbox folder.

So, anyway, when I looked in couch.stderr, as suggested, I noticed this error repeated:

``./bin/couchdb: eval: line 355: syntax error near unexpected token `('``

I realized what it meant. Simply moving the hoodie app to a folder without spaces in the path made it immediately work.

## Error Message

See above :)

* Type: bug
* Component: server
* Impact: few
* Severity: broken expected use
* Requirements: 

Hi to my Hoodie friends! I love you folks! :)

**Comment by janl** [comment]
This is a known limitation in CouchDB, we can’t do much about this at this time. It might get fixed with CouchDB 2.0, which has a non-Autotools based build system. CouchDB 1.x inherits this from Autotools
